### PR TITLE
Bebop 2 toolchain

### PR DESCRIPTION
**Pull request by bartslinger**
Using `arm-2016.03-linaro` instead of `arm-2012.02`

Also excluded a jpeg header when jpeg is not found. Otherwise it won't compile.

### CODE REVIEW

**Comment by bartslinger** [comment]
I think travis fails because it cannot find the jpeg lib?

**Comment by fvantienen** [comment]
Strange including the header should not matter. Maybe it has something todo with the libjpeg version?

**Comment by fvantienen** [comment]
See pull request #5 which fixes the libjpeg problem for OSX

File: tuv_lib/CMakeLists.txt [hunk 1]
```diff
@@ -61,7 +61,10 @@ file(REMOVE "include/tuv/tuv.h")
 file(GLOB_RECURSE HDRS_INC RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/include"  "include/*.h")
 
 foreach(filename ${HDRS_INC})
-    file(APPEND "include/tuv/tuv.h" "#include <${filename}>\r\n")
+    if ("tuv/encoding/encoder_jpeg.h" STREQUAL ${filename} AND NOT ${JPEG_FOUND})
```
Comment (fvantienen) [inline tuv_lib/CMakeLists.txt]: This is not needed anymore due to #5

**Comment by bartslinger** [comment]
Already fixed in https://github.com/tudelft/tudelft_vision/pull/7

**Pull request closed without merging**